src/toil_scripts/adam_pipeline/adam_preprocessing.py

```diff
@@ -118,7 +121,10 @@ def call_adam(masterIP, inputs, arguments):
                 tool = "quay.io/ucsc_cgl/adam:962-ehf--6e7085f8cac4b9a927dc9fb06b48007957256b80",
                 docker_parameters = masterIP.docker_parameters(["--net=host"]),
                 tool_parameters = default_params + arguments,
-                sudo = inputs['sudo'])
+                inputs=[],
```

Comment:
[] and {} should be the default of inputs and outputs, respectively. 

But do read: http://docs.python-guide.org/en/latest/writing/gotchas/